OWNERS_ALIASES

```diff
@@ -1,9 +1,5 @@
 # See the OWNERS docs: https://go.k8s.io/owners
 aliases:
   metrics-server-approvers:
-  - s-urbaniak
```

Comment:
I think we should leave repo admins (sig leaders) and repo writers as approvers
Sorry for making mistake in https://github.com/kubernetes/org/issues/1382 and will update it to match approvers.

Reply:
kk, will update quick

Reply:
Actually that leaves it the same as it was lol. I'll just go ahead and close the PR and call it done.

Reply:
Should `owners` section in `OWNERS` be removed?